org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/UiTextContributor.java

```diff
@@ -30,6 +30,7 @@ public void contribute(Set<String> textKeys) {
         "ColumnSorting",
         "CorrelationId",
         "DateIsNotAllowed",
+        "DefaultValue",
```

Comment:
Where is this text used?

Reply:
moved to the layer where it's actually used